Description: fix man pages
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Last-Update: 2024-03-10
Applied-Upstream: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ebfc/ebfc.1
+++ b/ebfc/ebfc.1
@@ -1,5 +1,4 @@
 .TH EBFC 1 "May 2021" "ELF kickers 3.2"
-.LO 1
 .SH NAME
 ebfc \- ELF Brainfuck compiler
 .SH SYNOPSIS
--- a/elfls/elfls.1
+++ b/elfls/elfls.1
@@ -1,5 +1,4 @@
 .TH ELFLS 1 "May 2011" "ELF kickers 3.0"
-.LO 1
 .SH NAME
 elfls \- display the contents of an ELF file
 .SH SYNOPSIS
--- a/elftoc/elftoc.1
+++ b/elftoc/elftoc.1
@@ -1,5 +1,4 @@
 .TH ELFTOC 1 "May 2011" "ELF kickers 3.0"
-.LO 1
 .SH NAME
 elftoc \- translate an ELF file into a C struct
 .SH SYNOPSIS
--- a/infect/infect.1
+++ b/infect/infect.1
@@ -1,5 +1,4 @@
 .TH INFECT 1 "May 2011" "ELF kickers 3.0"
-.LO 1
 .SH NAME
 infect \- insert malicious code into an existing binary
 .SH SYNOPSIS
--- a/objres/objres.1
+++ b/objres/objres.1
@@ -1,5 +1,4 @@
 .TH OBJRES 1 "May 2011" "ELF kickers 3.0"
-.LO 1
 .SH NAME
 objres \- compile binary data into an object file.
 .SH SYNOPSIS
--- a/rebind/rebind.1
+++ b/rebind/rebind.1
@@ -1,5 +1,4 @@
 .TH REBIND 1 "May 2011" "ELF kickers 3.0"
-.LO 1
 .SH NAME
 rebind \- modify the binding of global variables in an ELF object file
 .SH SYNOPSIS
--- a/sstrip/sstrip.1
+++ b/sstrip/sstrip.1
@@ -1,5 +1,4 @@
 .TH SSTRIP 1 "May 2011" "ELF kickers 3.0"
-.LO 1
 .SH NAME
 sstrip \- truncates ELF files to remove non-program content.
 .SH SYNOPSIS
